tests/components/signal_messenger/test_notify.py

```diff
@@ -0,0 +1,122 @@
+"""The tests for the signal_messenger platform."""
+
+import os
+import tempfile
+import unittest
+from unittest.mock import patch
+
+from pysignalclirestapi import SignalCliRestApi
+import requests_mock
+
+import homeassistant.components.signal_messenger.notify as signalmessenger
+from homeassistant.setup import async_setup_component
+
+BASE_COMPONENT = "notify"
+
+
+async def test_signal_messenger_init(hass):
+    """Test that service loads successfully."""
+
+    config = {
+        BASE_COMPONENT: {
+            "name": "test",
+            "platform": "signal_messenger",
+            "url": "http://127.0.0.1:8080",
+            "number": "+43443434343",
+            "recipients": ["+435565656565"],
+        }
+    }
+
+    with patch("pysignalclirestapi.SignalCliRestApi.send_message", return_value=None):
+        assert await async_setup_component(hass, BASE_COMPONENT, config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4929115)